lib/card.rb

```diff
@@ -0,0 +1,10 @@
+class Card
+  attr_reader :suit,
+                :value,
+                :rank
```

Comment:
Very minor nit, but if you're going to put attr_readers on separate lines, they should all be aligned, so I'd scoot `:value` and `:rank` back two spaces to line everything up.